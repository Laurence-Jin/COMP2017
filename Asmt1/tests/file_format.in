Wirragulla:::Cardiff::09:38:23
Wirragulla::Cardiff::10:01:23:
Wirragulla::Cardiff:::10:3:23
Wirragulla::Cardiff::1:59:23
Wirragulla::Cardiff::11:01:23