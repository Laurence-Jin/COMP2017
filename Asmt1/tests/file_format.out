Invalid Colon Format In File
